### ISSUE TITLE
Detect suspend / resume events

### ISSUE THREAD

**Issue by scoddy** [fd0/grobi#7]
It would be very helpful if grobi could react to resume events, e.g. when resuming from a different state (docked / undocked) as the laptop was suspended in. 

Probably from systemd/dbus?

**Comment by fd0** [comment]
I don't think this is necessary. If the display configuration was changed during suspend, grobi will be notified by the XRANDR change events it subscribed to. Other events are not interesting IMHO. 
I'm going to close this issue for now, please feel free to add further comments. Thanks!
